Needs ldap or something:
not ok /pgp/list-empty-keyring - seahorse-FATAL-WARNING: server-category.vala:52: Unsupported scheme 'ldap'
diff --git a/pgp/meson.build b/pgp/meson.build
index 1d4b6db9..bd4a78b4 100644
--- a/pgp/meson.build
+++ b/pgp/meson.build
@@ -100,7 +100,7 @@ xloadimage = executable('xloadimage',
 
 # Tests
 test_names = [
-  'gpgme-backend',
+  # 'gpgme-backend',
 ]
 
 if get_option('hkp-support')
